packages/donations-app/app/components/checkout/checkoutController.js

```diff
@@ -1,69 +1,34 @@
 angular.module('aliceApp')
-  .controller('CheckoutController', ['$rootScope', '$uibModal', '$http', '$stateParams', '$state', '$sce', '$timeout', 'AuthService', 'ProjectService', 'NotificationService', 'API', '$scope', 'MANGO', function ($rootScope, $uibModal, $http, $stateParams, $state, $sce, $timeout, AuthService, ProjectService, NotificationService, API, $scope, MANGO) {
+  .controller('CheckoutController', ['$rootScope', '$uibModal', '$http', '$stateParams', '$state', '$sce', '$timeout', 'AuthService', 'ProjectService', 'NotificationService', 'API', '$scope', 'MANGO', 'CheckoutService', function ($rootScope, $uibModal, $http, $stateParams, $state, $sce, $timeout, AuthService, ProjectService, NotificationService, API, $scope, MANGO, CheckoutService) {
 
-    // Set MANGOPAY API base URL and Client ID
-    mangoPay.cardRegistration.baseURL = MANGO.url;
-    mangoPay.cardRegistration.clientId = MANGO.clientId;
-
-    var amounts = {
+    const amounts = {
       amount10: 1000,
-      amount25: 2500,
-      amount50: 5000,
-      amount100: 10000,
+      amount20: 2000,
+      amount80: 8000,
+      amount150: 15000,
       amountOther: 0
     };
 
-
     var vm = this;
     vm.validatedGuest = false;
     vm.guest = {};
     vm.auth = AuthService;
-    vm.mode = 'CARD'; // enum: ["CARD", "3DS", "BANK_TRANSFER", "BANK_TRANSFER_REQUESTED"]
-
-    vm.card = {};
-
-    if (MANGO.url == 'https://api.sandbox.mangopay.com') {
-      vm.card = {
-        number: "3569990000000132",
-        expiryDate: "11/20",
-        cvc: "123",
-        name: AuthService.getLoggedFullName()
-      };
-    }
-
-    ProjectService.getProjectDetails($stateParams.projectCode).then(function (project) {
-      vm.project = project.data;
-    });
 
-    if (AuthService.getLoggedUser()) {
-      vm.card.name = AuthService.getLoggedUser().firstName + " " + AuthService.getLoggedUser().lastName;
-      vm.user = AuthService.getLoggedUser();
-    }
+    //TODO: Consider binding that to CheckoutService.donation.type, after we've got the bank mode
+    vm.mode = 'CARD';  // enum: ["CARD", "3DS", "BANK_TRANSFER", "BANK_TRANSFER_REQUESTED"]
```

Comment:
I think it's a good idea to bind it to CheckoutService.donation.type so the checkoutService will be the single source of truth

Reply:
I'm leaving that until we decide how to implement the bank transfer. 
Currently we use only a single mode (3ds is in different component), so we we move bank transfer to another component this variable will be removed.